### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

Note: I've filed some issues and made some PRs on some of the pkg_ items changed below, so they may technically not be incorrect today, but I believe that the changes should be made both here and upstream.

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@jsoref Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-common&pullRequestId=302)!

[Click here](https://cla.pivotal.io/sync/rabbitmq?repositoryId=rabbitmq/rabbitmq-common&pullRequestId=302) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@jsoref Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/rabbitmq?repositoryId=rabbitmq/rabbitmq-common&pullRequestId=302)!

File: LICENSE-MPL-RabbitMQ [hunk 1]
```diff
@@ -327,7 +327,7 @@
      shall survive.
 
      8.2.  If You initiate litigation by asserting a patent infringement
-     claim (excluding declatory judgment actions) against Initial Developer
+     claim (excluding declaratory judgment actions) against Initial Developer
```
Comment (jsoref) [inline LICENSE-MPL-RabbitMQ]: Oh, I should call this one out. This is a license file, lawyers typically have strong opinions about them. Generally, dictionaries prefer the spelling I'm suggesting. But, I'd certainly understand people objecting, and dropping this change (or any other) isn't a problem.

File: src/supervisor2.erl [hunk 2]
```diff
@@ -1438,7 +1438,7 @@ validMods(Mods) -> throw({invalid_modules, Mods}).
 %%% Add a new restart and calculate if the max restart
 %%% intensity has been reached (in that case the supervisor
 %%% shall terminate).
-%%% All restarts accured inside the period amount of seconds
+%%% All restarts accrued inside the period amount of seconds
```
Comment (jsoref) [inline src/supervisor2.erl]: I'm not sure about this

File: src/supervisor2.erl [hunk 3]
```diff
@@ -1085,7 +1085,7 @@ monitor_child(Pid) ->
 %%
 %% Shutdown all dynamic children. This happens when the supervisor is
 %% stopped. Because the supervisor can have millions of dynamic children, we
-%% can have an significative overhead here.
+%% can have significant overhead here.
```
Comment (jsoref) [inline src/supervisor2.erl]: I'd never seen this word before, it's apparently a word, that said, it almost certainly wouldn't be compatible w/ `an`, and I don't think it's the right word...

File: src/gen_server2.erl [hunk 4]
```diff
@@ -1275,7 +1275,7 @@ name_to_pid(Name) ->
         undefined ->
             case whereis_name(Name) of
                 undefined ->
-                    exit(could_not_find_registerd_name);
+                    exit(could_not_find_registered_name);
```
Comment (jsoref) [inline src/gen_server2.erl]: I'm not sure what erlang is doing here, so this change might be incorrect...

Comment (michaelklishin) [reply src/gen_server2.erl]: It's just a reason the process terminates with. I doubt it's a common scenario, there are two call sites that use this atom across all repos (both times in `rabbitmq_common` while exiting, enver handling exits).

Let's leave this in.

File: src/supervisor2.erl (hunk 3 above)
Comment (michaelklishin) [reply src/supervisor2.erl]: I agree, the intent is to say "significant overhead".

File: src/supervisor2.erl (hunk 2 above)
Comment (michaelklishin) [reply src/supervisor2.erl]: That sentence is broken in more than one way. I think it should say "all restarts occur".

File: LICENSE-MPL-RabbitMQ (hunk 1 above)
Comment (michaelklishin) [reply LICENSE-MPL-RabbitMQ]: 👍

File: src/supervisor2.erl (hunk 2 above)
Comment (jsoref) [reply src/supervisor2.erl]: I've changed it to occurring in 2ff7143

File: src/gen_server2.erl (hunk 4 above)
Comment (jsoref) [reply src/gen_server2.erl]: I can't tell what you mean by "leave this in" -- you could mean "leave the existing code in" or "leave the proposed replacement code in"

Comment (michaelklishin) [reply src/gen_server2.erl]: Leave the proposed replacement.

**Pull request merged**